### PR TITLE
#[UIC-1037] Enabled RVF: Dev setup using Pyenv | pipenv | pipfile | pipfile.lock

### PR DESCRIPTION
**Pull request by harshit-kumar-guavus**
Description:

    1. Adding packages in Pipfile

    2. Deleted requirement.txt and requirement-dev.txt files

    3. Corresponding changes in setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def get_git_sha():
         'requests-kerberos==0.12.0',
         'django==2.1.5',
         'wtforms-json',
-        'pyhive>=0.6.1',
+        'pyhive<=0.6.1',
```
Comment (bipinsoniguavus) [inline setup.py]: why we have put `<` instead?

File: setup.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def get_git_sha():
         'contextlib2',
         'elasticsearch',
         'croniter==0.3.25',
-        'cryptography>=2.4.2',
+        'cryptography<=2.4.2',
```
Comment (bipinsoniguavus) [inline setup.py]: why we have put `<` instead?

File: Pipfile [hunk 3]
```diff
@@ -100,9 +105,9 @@ SQLAlchemy-Utils = "==0.32.21"
 SQLAlchemy = "==1.2.2"
 thrift_sasl = "==0.3.0"
 pipenv = "*"
-python-ldap = "==3.1.0"
```
Comment (bipinsoniguavus) [inline Pipfile]: Why we have removed this lib?

File: setup.py (hunk 2 above)
Comment (harshit-kumar-guavus) [reply setup.py]: so that it gets aligned with version mentioned in  Pipfile

File: setup.py (hunk 1 above)
Comment (harshit-kumar-guavus) [reply setup.py]: so that it gets aligned with version mentioned in  Pipfile

File: Pipfile (hunk 3 above)
Comment (harshit-kumar-guavus) [reply Pipfile]: due to duplicacy in  Pipfile i.e. it was mentioned 2 times in same Pipfile

File: setup.py (hunk 2 above)
Comment (bipinsoniguavus) [reply setup.py]: As discussed, please revert to original, i.e. `cryptography>=2.4.2`

File: setup.py (hunk 1 above)
Comment (bipinsoniguavus) [reply setup.py]: As discussed, please revert to original, i.e. `pyhive>=0.6.1`

**Pull request merged**